model.py

```diff
@@ -615,6 +617,32 @@ def update(self, start, end, position):
         UniqueConstraint('patron_id', 'license_pool_id'),
     )
 
+class Annotation(Base):
+    LS_NAMESPACE = u"http://librarysimplified.org/terms/annotation/"
+
+    IDLING = LS_NAMESPACE + u'idling'
+
+    MOTIVATIONS = [
+        IDLING
+    ]
+
+    __tablename__ = 'annotations'
+    id = Column(Integer, primary_key=True)
+    patron_id = Column(Integer, ForeignKey('patrons.id'), index=True)
+    identifier_id = Column(Integer, ForeignKey('identifiers.id'), index=True)
+    motivation = Column(
+        Enum(*MOTIVATIONS, name="motivation"),
+        index=True
```

Comment:
I'll change it. Even if we only ever allow a small number, we won't need migrations as we add the others.